src/browser/custom.d.ts

```diff
@@ -2,3 +2,37 @@ declare module '*.svg' {
   const content: string
   export default content
 }
+
+declare module 'react-suber' {
+  interface BusProps {
+    bus: Bus
+  }
+  const withBus: (comp: any) => React.ComponentType
+  const BusProvider: React.ComponentType
+  export { withBus, BusProvider, BusProps }
+}
+
+declare module 'suber' {
```

Comment:
This is a start for typing up suber and react-suber. Some of the `any`s are unavoidable sadly. Perhaps we could do something with generics but not sure if it's worth it.